.changeset/tough-plants-divide.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-catalog-react': patch
+---
+
+Added missing items to `overridableComponents`
+
+**BREAKING** Overridable component name for styling `BackstageMissingAnnotationEmptyState` has been changed to `CatalogReactMissingAnnotationEmptyState`
```

Comment:
The other breaking changes I think are fine, but this one is a bit trickier, we don't really want to have to bump catalog to 2.0

I think the sensible thing to do here is probably to just leave it as is for now?

Reply:
Although now when looking at the changes it seems like this might not be breaking at all, just added?

Reply:
OK, had to dig into this, down side of long lived PRs. So `MissingAnnotationEmptyState` was moved from `@backstage/core-components` to here in: https://github.com/backstage/backstage/pull/20842 but wasn't added to the `overridableComponents` in that PR. Given that you are right this isn't breaking as it is finally being added.

Reply:
I've deleted the changeset seeing as it not needed and added `@backstage/plugin-catalog-react` to the generic changeset.